src/pages/Proposal/index.tsx

```diff
@@ -53,6 +57,19 @@ export const ProposalUi: React.FC<ProposalUiProps> = ({
     <div css={styles.root}>
       <ProposalSummary css={styles.summary} proposal={proposal} />
 
+      {!voteProposalFeatureEnabled && (
+        <div className="mb-8 w-full" data-testid={TEST_IDS.votingDisabledWarning}>
+          <NoticeInfo
+            title={t('vote.multichain.votingOnlyEnabledOnBsc')}
+            description={
+              <Button variant="text" onClick={() => switchChain({ chainId: chains[0].id })}>
```

Comment:
The web3 client now exports the Governance chain for the current environment:

```suggestion
import { governanceChain } from 'clients/web3';

              <Button variant="text" onClick={() => switchChain({ chainId: governanceChain.id })}>
```